nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md

```diff
@@ -40,6 +41,7 @@ partial:disableautosubscribe
 
 ## Manually subscribing to a message
```

Comment:
I'm considering to extend this section a little bit about the usable APIs to subscribe/unsubscribe (`IBus` in v5 but only on the `IMessageSession` in v6, so no longer possible in message handlers). Thoughts?